src/main/java/uk/gov/hmcts/reform/unspec/service/docmosis/sealedclaim/SealedClaimFormGenerator.java

```diff
@@ -76,8 +79,12 @@ public SealedClaimForm getTemplateData(CaseData caseData) {
             .referenceNumber(caseData.getLegacyCaseReference())
             .issueDate(caseData.getClaimIssuedDate())
             .submittedOn(caseData.getClaimSubmittedDateTime().toLocalDate())
-            .claimantExternalReference(caseData.getSolicitorReferences().getApplicantSolicitor1Reference())
-            .defendantExternalReference(caseData.getSolicitorReferences().getRespondentSolicitor1Reference())
+            .claimantExternalReference(solicitorReferences
+                                           .map(SolicitorReferences::getApplicantSolicitor1Reference)
```

Comment:
should this not be something like "not provided"

Reply:
Nope that's fine.

Reply:
"not provided" was for COS requirements and not for sealed claim.